selendroid-standalone/src/test/java/io/selendroid/standalone/server/model/DeviceStoreFixture.java

```diff
@@ -32,6 +34,7 @@ protected static DefaultHardwareDevice anDevice(String name, DeviceTargetPlatfor
     DefaultHardwareDevice device = mock(DefaultHardwareDevice.class);
     when(device.getModel()).thenReturn(name);
     when(device.getTargetPlatform()).thenReturn(platform);
+    when(device.getModel()).thenReturn("Nexus 5");
```

Comment:
@asm89 
I found that it is unnecessary to make that mock call (it is removed) since the original code already returned <code>name</code> with <code>getModel()</code>.